Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/light_rifle.yml

```diff
@@ -1,6 +1,6 @@
 - type: entity
   id: BulletLightRifleRubber
-  name: bullet (.20 rifle rubber)
+  name: bullet (.30 rifle rubber)
```

Comment:
light rifle is 20 not 30 mustve made this mistake when moving these to deltav folder, flip them around and have the rubber change in the other file

Reply:
https://github.com/DeltaV-Station/Delta-v/blob/cd949addd18e6a9a1d19544fb7884960ab210aeb/Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Boxes/light_rifle.yml#L4-L5

regular rifle is .20
https://github.com/DeltaV-Station/Delta-v/blob/cd949addd18e6a9a1d19544fb7884960ab210aeb/Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Boxes/rifle.yml#L49-L50

Reply:
real